src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/main.py

```diff
@@ -204,54 +199,37 @@ def _store_extension_parameters(parameter_store, config):
 
 
 # pylint: disable=too-many-locals
-def worker_thread(
-    account_id,
-    sts,
-    config,
-    s3,
-    cache,
-    updated_kms_bucket_dict
-):
+def worker_thread(account_id, sts, config, s3, cache, updated_kms_bucket_dict):
     """
     The Worker thread function that is created for each account
     in which CloudFormation create_stack is called
     """
     LOGGER.debug("%s - Starting new worker thread", account_id)
 
-    organizations = Organizations(
-        role=boto3,
-        account_id=account_id
-    )
+    organizations = Organizations(role=boto3, account_id=account_id)
     ou_id = organizations.get_parent_info().get("ou_parent_id")
 
     account_path = organizations.build_account_path(
-        ou_id,
-        [],  # Initial empty array to hold OU Path,
-        cache
+        ou_id, [], cache  # Initial empty array to hold OU Path,
```

Comment:
Can you revert this change, as the comment is linked to the empty list. Not the cache or all three arguments together?